examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.php

```diff
@@ -0,0 +1,602 @@
+                ])
+            ])
+        );
+
+        $mutateOperations[] = self::createMutateOperationForUnit(
+            $customerId,
+            $assetGroupId,
+            $tempId--,
+            $conditionOtherSubdivisionId,
+            new ListingGroupFilterDimension([
+                'product_brand' => new ListingGroupFilterDimension\ProductBrand()
+            ])
+        );
+
+        // Issues a mutate request to create everything and prints its information.
+        $googleAdsServiceClient = $googleAdsClient->getGoogleAdsServiceClient();
+        $response = $googleAdsServiceClient->mutate(
+            $customerId,
+            $mutateOperations
+        );
+
+        self::printResponseDetails($mutateOperations, $response);
+        // [END add_performance_max_product_listing_group_tree_1]
+    }
+
+    /**
+     * Fetches all of the asset group listing group filters in an asset group.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param int $assetGroupResourceName the resource name of the asset group
+     * @return AssetGroupListingGroupFilter[] the list of asset group listing group filters
+     */
+    // [START add_performance_max_product_listing_group_tree_7]
+    private static function getAllExistingListingGroupFilterAssetsInAssetGroup(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $assetGroupResourceName
+    ): array {
+        $googleAdsServiceClient = $googleAdsClient->getGoogleAdsServiceClient();
+        // Creates a query that retrieves asset group listing group filters.
+        $query = sprintf(
+            'SELECT asset_group_listing_group_filter.resource_name, '
+            . 'asset_group_listing_group_filter.parent_listing_group_filter '
+            . 'FROM asset_group_listing_group_filter '
+            . 'WHERE asset_group_listing_group_filter.asset_group = "%s"',
+            $assetGroupResourceName
+        );
+
+        // Issues a search request by specifying page size.
+        $response =
+            $googleAdsServiceClient->search($customerId, $query, ['pageSize' => self::PAGE_SIZE]);
+
+        $assetGroupListingGroupFilters = [];
+        // Iterates over all rows in all pages to get an asset group listing group filter.
+        // The limit to the number of listing group filters permitted in a Performance
+        // Max campaign can be found here:
+        // https://developers.google.com/google-ads/api/docs/best-practices/system-limits.
+        foreach ($response->iterateAllElements() as $googleAdsRow) {
+            /** @var GoogleAdsRow $googleAdsRow */
+            $assetGroupListingGroupFilters[] = $googleAdsRow->getAssetGroupListingGroupFilter();
+        }
+        return $assetGroupListingGroupFilters;
+    }
+    // [END add_performance_max_product_listing_group_tree_7]
+
+    /**
+     * Removes an existing tree of asset group listing group filters.
```

Comment:
It does not remove them, it only creates the mutate operations to do so.

Please replace with `Creates mutate operations for removing ...`